client/sites-dashboard-v2/dotcom-style.scss

```diff
@@ -6,6 +6,14 @@
 // Add new Dotcom specific styles to this file.
 .wpcom-site .layout__primary .main {
 	padding-bottom: 0;
+	--color-accent: #3858e9;
+	--color-accent-40: #3858e9;
+	--color-accent-60: #3858e9;
+	--color-link: #3858e9;
+	--color-primary: #3858e9;
+	--color-primary-50: #3858e9;
+	--color-primary-60: #3858e9;
+	--wp-admin-theme-color: #3858e9;
```

Comment:
Just two thoughts I had - would it not be better to extract these into their own colour scheme and then fill out all the variables? At the moment, quite a few variables aren't affected because this doesn't hit hover/focus styles.

<img width="288" alt="Screenshot 2024-05-02 at 08 01 59" src="https://github.com/Automattic/wp-calypso/assets/43215253/08a934ff-18e1-4f0a-a750-b2664dd2b1a6">
<img width="450" alt="Screenshot 2024-05-02 at 08 02 22" src="https://github.com/Automattic/wp-calypso/assets/43215253/b3bc77b5-9d05-4e93-961a-ccdafd8cfb51">
<img width="478" alt="Screenshot 2024-05-02 at 08 05 07" src="https://github.com/Automattic/wp-calypso/assets/43215253/dbf512f5-14a3-4e7f-90db-1b3ee813d102">

And secondly, by forcing them to be the same colour, the hover/focus differentiations are lost - for example, in the Nav Selector above - which feels a little odd. Is that intentional?

Reply:
Thanks @Aurorum - I agree about building a color scheme for the dashboard. Overriding the colors feels flakey and likely to break.

@davemart-in - Do we have a hover color to work with? If you look at the video in the description, we typically have one for other themes but I haven't seen one with the blueberry color.

Reply:
@lucasmendes-design @nuriapenya do either of you know if we have a blueberry hover color?

Reply:
Just noting that the hover colours are fixed now that it's directly using the Modern scheme. :)